externs/shaka/player.js

```diff
@@ -90,6 +90,7 @@ shaka.extern.StateChange;
  *   The height of the current video track.
  * @property {number} streamBandwidth
  *   The bandwidth required for the current streams (total, in bit/sec).
+ *   It has into account the playbackrate.
```

Comment:
Sorry, small English nit: You "take" something into account, rather than "have" it into account.